source/App.cpp

```diff
@@ -87,34 +86,42 @@ void App::onInit() {
 	updateMouseSensitivity();			// Update (apply) mouse sensitivity
 	updateSessionDropDown();			// Update the session drop down to remove already completed sessions
 	updateSessionPress();				// Update session to create results file/start collection
+
+	setFrameDuration(m_wallClockTargetDuration, REAL_TIME);
 }
 
 /** Handle then user settings window visibility */
 void App::openUserSettingsWindow() {
-    m_userSettingsMode = true;
-    m_userSettingsWindow->setVisible(m_userSettingsMode);
+	m_userSettingsMode = true;
+	m_userSettingsWindow->setVisible(m_userSettingsMode);
+}
+
+/** Handle then user settings window visibility */
```

Comment:
Is "then" a typo for "the"? Also, I don't think this function is needed, particularly because I believe `m_userSettingsMode` is going away in #108 .

Reply:
Looks like it. I can change it. The tests will need some way to close the settings window, but it could inject Esc just as well.

Reply:
I believe you can call `m_userSettingsWindow->setVisible(false);` to hide the window. Might still need the `m_userSettingsMode = false;` in the current version, though that line can go away once the other PR is merged.

If that's not doable, then I can see why you might need the callback function and how that would be cleaner for the test.

Reply:
Technically speaking we don't really need `m_userSettingsMode` we can just use the visible flag (`m_userSettingsWindow->visible()`) for the window to manage whether we are in this mode.

For example `m_userSettingsWindow->setVisible(!m_userSettingsWindow->visible())` would suffice.